README.md

```diff
@@ -91,13 +110,14 @@ AKLIMATE and subSCOPE do not need manual model data download.
 ### 1C. Feature Renaming Reference Files
 Download and decompress the reference files - renaming any user data feature to nomenclature that machine learning models will recognize (TMP nomenclature).
 
-The `ft_name_convert.tar.gz` file (SynapseID syn51315102) can be downloaded from the Publication Page and then placed in `tools/`
+The `ft_name_convert.tar.gz` file can be downloaded from the Publication Page and then placed in `tools/`
 ```
 cd tools
 tar -xzf ft_name_convert.tar.gz
 cd ..
 ```
 
+
 # Data Requirements
 User input data must be in tab separated format.
```

Comment:
Samples as rows, features as columns; first column is Labels.

Reply:
good point, updated in commit [d24da3e](https://github.com/NCICCGPO/gdan-tmp-models/pull/54/commits/d24da3ef436a4ee4040cdaf53c2c2bdd3532c46c)